sql/moz-fx-data-shared-prod/firefox_accounts/docker_fxa_customs_sanitized/view.sql

```diff
@@ -2,6 +2,105 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_accounts.docker_fxa_customs_sanitized`
 AS
 SELECT
-  *
+  `date`,
+  logName,
+  (
+    SELECT AS STRUCT
+      resource.type,
+      (
+        SELECT AS STRUCT
+          resource.labels.instance_id,
+          resource.labels.zone,
+          resource.labels.project_id,
+          CAST(NULL AS STRING) AS cluster_name,
+          CAST(NULL AS STRING) AS namespace_name,
+          CAST(NULL AS STRING) AS container_name,
+          CAST(NULL AS STRING) AS pod_name,
+          CAST(NULL AS STRING) AS location
+      ) AS labels
+  ) AS resource,
+  textPayload,
+  (
+    SELECT AS STRUCT
+      jsonPayload.block,
+      jsonPayload.op,
+      jsonPayload.action
+  ) AS jsonPayload,
+  `timestamp`,
+  receiveTimestamp,
+  severity,
+  insertId,
+  httpRequest,
+  (
+    SELECT AS STRUCT
+      labels.compute_googleapis_com_resource_name,
+      CAST(NULL AS STRING) AS k8s_pod_deployment,
+      CAST(NULL AS STRING) AS k8s_pod_app_kubernetes_io_component,
+      CAST(NULL AS STRING) AS k8s_pod_env_code,
+      CAST(NULL AS STRING) AS k8s_pod_pod_template_hash,
+      CAST(NULL AS STRING) AS k8s_pod_app_kubernetes_io_name,
+      CAST(NULL AS STRING) AS k8s_pod_app_kubernetes_io_instance,
+  ) AS labels,
+  operation,
+  trace,
+  spanId,
+  traceSampled,
+  sourceLocation,
+  split,
 FROM
   `moz-fx-data-shared-prod.firefox_accounts_derived.docker_fxa_customs_sanitized_v1`
+  -- this filter indicates up until when the source table is expected to contains events
+  -- due to AWS to GCP migration, after this date we expect the v2 to contain all the events from GCP.
+  WHERE `date` < "2023-09-15"
```

Comment:
Looks like the v1 ETL is still pulling in some data beyond 2023-09-14.  Should that data be included as well?

And should the v1 ETL be descheduled?

Reply:
yes, the ETL should be descheduled.

Reply:
What about the data that's still showing up in the v1 ETL?  If the migration is actually complete, why is that data showing up?

Reply:
hmm, I see that too. I will try and confirm with jbuck and in the interest of time I added back the v1 to the ETL and will create an action item to deschedule it once we're confident no new data is arriving from the AWS based deployment.